tests/framework/e2e/e2e.go

```diff
@@ -87,7 +86,7 @@ func (e e2eRunner) NewCluster(ctx context.Context, t testing.TB, opts ...config.
 	default:
 		t.Fatalf("PeerTLS config %q not supported", cfg.PeerTLS)
 	}
-	epc, err := NewEtcdProcessCluster(ctx, t, &e2eConfig)
+	epc, err := NewEtcdProcessCluster(ctx, t, e2eConfig)
```

Comment:
Can we update it for now? Eventually we will get rid of the third parameter `nil`.
```suggestion
	epc, err := NewEtcdProcessCluster(ctx, t, nil, WithConfig(&e2eConfig))
```

Reply:
The next PR will replaces ALL initialisations of `EtcdProcessClusterConfig` and remove the `nil` parameter.